packages/context/test/unit/binding.ts

```diff
@@ -98,6 +98,31 @@ describe('Binding', () => {
     });
   });
 
+  describe('toJSON()', () => {
+    it('converts to plain JSON object', () => {
+      const json = binding.toJSON();
+      expect(json).to.eql({
+        key: key,
+        scope: BindingScope.TRANSIENT,
+        tags: [],
+        isLocked: false,
+      });
+    });
+
+    it('converts to plain JSON object', () => {
```

Comment:
This test has the same name as the previous test. Please use unique test names.

I am proposing to keep the test name here as it is now, and remove the previous test completely, as it does not seem to add any new value.